src/applications/disability-benefits/all-claims/submit-transformer.js

```diff
@@ -318,6 +320,43 @@ export function transform(formConfig, form) {
     return clonedData;
   };
 
+  // transform secondary disabilities into primary, with description appended
+  const transformSecondaryDisabilities = formData => {
+    // Skip transform in production environment
+    if (environment.isProduction() || !formData.newSecondaryDisabilities) {
+      return formData;
+    }
+
+    const clonedData = _.cloneDeep(formData);
+
+    const transformedSecondaries = clonedData.newSecondaryDisabilities.map(
+      sd => {
+        // prepend caused by condition to primary description
+        const descString = [
+          'Secondary to ',
+          sd.causedByDisability,
+          '\n',
+          sd.causedByDisabilityDescription,
+        ].join('');
+
+        return {
+          condition: sd.condition,
+          cause: causeTypes.NEW,
+          classificationCode: sd.classificationCode,
+          // truncate description to 400 characters
+          primaryDescription: descString.substring(0, 400),
```

Comment:
:thinking: This would cut off some of the description that the veteran wrote, right? Is there maybe a way to truncate a long `causedByDisability` first?

I guess I'm not really sure which would be more important to a rater.